pinot-controller/src/test/java/com/linkedin/pinot/controller/helix/PinotResourceManagerTest.java

```diff
@@ -66,6 +69,13 @@ public void setUp() throws Exception {
     Assert.assertEquals(_helixAdmin.getInstancesInClusterWithTag(HELIX_CLUSTER_NAME, "DefaultTenant_REALTIME").size(),
         1);
 
+    // Verifying instance configs.
+    ZNRecord znRecord = _pinotHelixResourceManager.getCacheInstanceConfigsDataAccessor().get("/Server_localhost_0", null, AccessOption.PERSISTENT);
```

Comment:
You should invoke the other method in the test, and compare the result against getting the instanceconfig directly from helix.